src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5616,6 +5616,21 @@ public sealed partial class NotNullWhenAttribute : System.Attribute
         public NotNullWhenAttribute(bool returnValue) { }
         public bool ReturnValue { get { throw null; } }
     }
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Property, Inherited = false)]
+    [CLSCompliant(false)]
```

Comment:
Nit: Add the "System." namespace to these (see existing examples).  The tool that generates these files would do so, so without them this will just result in churn the next time the files are auto-gen'd.